addon/session-stores/adaptive.js

```diff
@@ -44,7 +46,21 @@ export default Base.extend({
     @default null
     @public
   */
-  cookieDomain: null,
+  _cookieDomain: null,
+  cookieDomain: computed({
+    get() {
+      return this.get('_cookieDomain');
+    },
+    /*jshint unused: true */
```

Comment:
Does this actually report an unused error for `key`? I don't think it should as we cannot prevent having it in the arguments list here. You could maybe replace it with `_` to make clear it's unused though.